js/models/conversations.js

```diff
@@ -2051,6 +2051,20 @@
       const endpoint = `${server}/channels/${channelId}/messages`;
       return endpoint;
     },
+    getChannelToken() {
+      if (!this.isPublic()) {
+        return null;
+      }
+      return this.get('channelToken');
+    },
+    async setChannelToken(token) {
+      if (!_.isEqual(this.get('channelToken'), token)) {
```

Comment:
Just curious, what does `_.isEqual` do differently than `==`/`===` ?

Reply:
Oh is it because `token` is array-like and every item needs to be compared one by one?